doc/tutorials/python/08-visualization/scripts/simulation.py

```diff
@@ -1,5 +1,5 @@
 from __future__ import print_function
-import espressomd._system as es
+
```

Comment:
Can you please just remove that empty lines?

Reply:
I usually leave a blank line after __future__ imports, as they are required to appear before any code (except docstrings) and in contrast to other imports are not script-specific -- just a part of the preamble.
I think this is also the way pep8 suggests (see https://www.python.org/dev/peps/pep-0008/#imports section 'Module level dunder names').

Reply:
We have bigger fish to fry...

Reply:
Thats why I just use autopep8 -i